portal-kernel/src/com/liferay/portal/kernel/util/Validator.java

```diff
@@ -15,6 +15,7 @@
 package com.liferay.portal.kernel.util;
 
 import com.liferay.petra.string.CharPool;
+import com.liferay.petra.string.StringBundler;
```

Comment:
@W2jx This change is kind of weird, I know the root reason you did it, but let's try to focus on current task by not modifying kernel's StringBundler

Reply:
@W2jx Additionally, please also skip other classes in portal-kernel that have copies in core/petra. They are going to be removed in the future.